vhs.go

```diff
@@ -98,8 +101,15 @@ func (vhs *VHS) Setup() {
 	vhs.CursorCanvas, _ = vhs.Page.Element("canvas.xterm-cursor-layer")
 
 	// Set Prompt
+	shellSetup := ""
+	switch vhs.Options.Shell {
+	case "", "bash":
+		shellSetup = fmt.Sprintf(` set +o history; unset PROMPT_COMMAND; export PS1="%s"; clear;`, vhs.Options.Prompt)
+	default:
```

Comment:
here I *think* we should probably add switches for the most common shells, at least zsh and fish, I would say

Reply:
Agreed! This is great!